src/lambda_alerting.py

```diff
@@ -3,41 +3,61 @@
 import boto3
 import logging
 
+from lib.core import time_utils
 from lib.aws.sqs_manager import SQSQueueSender
-from lib.aws.timestream_manager import TimestreamTableWriter
+from lib.aws.cloudwatch_manager import CloudWatchEventsPublisher
 from lib.event_mapper.aws_event_mapper import AwsEventMapper
 from lib.settings import Settings
 
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
-timestream_write_client = boto3.client("timestream-write")
 sqs_client = boto3.client("sqs")
+cloudwatch_client = boto3.client("logs")
 
 
-def write_event_to_timestream(records):
+def write_event_to_cloudwatch(
+    monitored_env_name: str,
+    resource_name: str,
+    service_name: str,
+    event_severity: str,
+    event: dict,
+):
     """
-    Writes a given list of records to an Amazon Timestream table.
+    Writes a given list of records to an Amazon CloudWatch logs.
 
-    Retrieves the database and table names from environment variables and uses
-    an instance of TimestreamTableWriter to write the provided records to the
-    specified Timestream table.
+    Retrieves the log group and log stream names from environment variables and uses
+    an instance of CloudWatchEventPublisher to write the provided records to the
+    specified CloudWatch log stream.
 
     Args:
-        records (list): A list of records to be written to the Timestream table.
+        monitored_env_name (str): The name of the monitored environment.
+        resource_name (str): The name of the AWS resource.
+        service_name (str): The name of the AWS service.
+        event_severity (str): Severity of the event.
+        event (dict): The event dict to be written to the CloudWatch stream.
 
     Returns:
         None: This function does not return anything but logs the outcome.
     """
-    db_name = os.environ["ALERT_EVENTS_DB_NAME"]
-    table_name = os.environ["ALERT_EVENTS_TABLE_NAME"]
+    log_group_name = os.environ["ALERT_EVENTS_CLOUDWATCH_LOG_GROUP_NAME"]
+    log_stream_name = os.environ["ALERT_EVENTS_CLOUDWATCH_LOG_STREAM_NAME"]
 
-    writer = TimestreamTableWriter(
-        db_name=db_name,
-        table_name=table_name,
-        timestream_write_client=timestream_write_client,
+    publisher = CloudWatchEventsPublisher(
+        log_group_name=log_group_name,
+        log_stream_name=log_stream_name,
+        cloudwatch_client=cloudwatch_client,
     )
-    result = writer.write_records(records=records)
+
+    logged_event = {}
+    logged_event["event"] = event
+    logged_event["monitored_environment"] = monitored_env_name
+    logged_event["resource_name"] = resource_name
+    logged_event["service_name"] = service_name
+    logged_event["event_severity"] = event_severity
+
+    logged_event_time = time_utils.iso_time_to_epoch_milliseconds(event["time"])
+    result = publisher.put_event(logged_event_time, json.dumps(logged_event, indent=-4))
```

Comment:
I'm just curious why indent value is negative? :)

Reply:
Was a typo which surprisingly did not affect the json document structure 😁 . Fixed 👍